### DISCUSSION TITLE
Change export directory for static HTML generation

### DISCUSSION THREAD

**Discussion by winston0410** [vercel/next.js#17737] [Help] [answered]
Is it possible to change the directory name for  static HTML generation in Next.js? I have checked the documentation, but I cannot find what I need.

**Comment by jamesmosier** [comment, marked as answer]
There is no way to rename this directory using Next. If you wanted to rename the directory you would have to run a custom script after your export completes.

One such Node method would be to use [`fs. renameSync`](https://nodejs.org/api/fs.html#fs_fs_renamesync_oldpath_newpath) after your build/export is completed.
